### DISCUSSION TITLE
Install.md is incorrect

### DISCUSSION THREAD

**Discussion by brplate** [opensourcepos/opensourcepos#4099] [Q&A]
Hi I'm a noob but can follow instructions :) as I am looking at this tool for my small business in any event, I have found that your installation instructions are not correct and appear to be missing steps. This is the link I used to download ospos [CLICK HERE](https://codeload.github.com/opensourcepos/opensourcepos/zip/refs/heads/master)

**Step3** in the install.md says _Execute the file `database/database.sql` to create the tables needed._

- There is no such file to be found.  It appears that the file name is tables.sql

![image](https://github.com/user-attachments/assets/79df8746-7b03-4735-bb79-6206375849ad)

**Step 4** say _Open `app/Config/config.php`_ but there is no config.php file
![image](https://github.com/user-attachments/assets/8e8918dd-4471-4f8a-9907-f3bc207c8e20)

I am next confused by **Step10** as there is no .env file in the app/Config directory.  The only .env file is found at the root of the ospos download

![image](https://github.com/user-attachments/assets/4b3c6bb9-113c-4e94-8ad0-0cc6affd14e1)

**Comment by jekkos** [comment]
Thanks for your comments. I'll update the instructions this weekend
